File: modul 5/soal 4/PRAK404-2210817220034-Wanda Karliyanti.c
#include <stdio.h>

int main (){

int p;
float a,b,hasil;
while (p!=5){

printf("Pilih program\n");
printf("1.Penjumlahan\n");
printf("2.Pengurangan\n");
printf("3.Perkalian\n");
printf("4.Pembagian\n");
printf("5.Exit\n");
printf("Masukkan Pilihan : ");
scanf("%d",&p);

    if (p==5){
        printf("Terima kasih, telah menggunakan kalkulator WANDA KARLIYANTI");}
    else if (p<0 || p>5){
        printf("Input anda salah, silahkan coba lagi\n\n");}
    else{
        printf("Masukkan Nilai pertama : ");scanf("%f",&a);

printf("Masukkan Nilai kedua : ");
scanf("%f",&b);
    if (p==1){hasil=a+b;
        printf("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f\n\n",a,b,hasil);}
    else if (p==2){hasil= a - b;
        printf("Hasil pengurangan antara %.2f dengan %.2f adalah %.2f\n\n",a,b,hasil);}
    else if (p==3){hasil= a * b;
        printf("Hasil perkalian antara %.2f dengan %.2f adalah %.2f\n\n",a,b,hasil);}
    else{hasil=a/b;
        printf("Hasil pembagian antara %.2f dengan %.2f adalah %2.f\n\n",a,b,hasil);}

}
}
}

File: modul 5/soal 4/PRAK404-2210817220034-Wanda Karliyanti.py
while (True):
    print("Pilih program")
    print("1. Penjumlahan")
    print("2. Pengurangan")
    print("3. Perkalian")
    print("4. Pembagian")
    print("5. Exit")
    p=float(input("Masukkan Pilihan : "))
    if p==5:break
    elif p>=6:
        print("Input anda salah, silahkan coba lagi\n\n")
        continue
    n1=float(input("Masukkan nilai pertama : "))
    n2=float(input("Masukkan nilai kedua : "))
    if p==1:
        hasil=(n1+n2)
        print("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f\n\n"%(n1, n2, hasil)) 
    elif p==2:
         hasil=(n1-n2) 
         print("Hasil pengurangan antara %.2f dengan %.2f adalah %.2f\n\n"%(n1, n2, hasil))
    elif p==3:
         hasil=(n1*n2) 
         print("Hasil perkalian antara %.2f dengan %.2f adalah %.2f\n\n"%(n1,n2, hasil)) 
    elif p==4:
         hasil=(n1/n2) 
         print("Hasil pembagian antara %.2f dengan %.2f adalah %.2f\n\n"%(n1, n2 , hasil))
print("Terima kasih, telah menggunakan kalkulator WANDA KALIYANTI")